### DISCUSSION TITLE
Will Courseplay Exist in FS25?

### DISCUSSION THREAD

**Discussion by Eliakim27** [Courseplay/Courseplay_FS22#3347] [General]
Hi there I create content for YouTube and my top mod would have to be Courseplay, in a jest on one Episode I said Giants should talk to the Courseplay Team and work it into the game, well, the confirmed features (for FS25) are a simplistic Courseplay (and they called it that) will you still be making a Design and Save your courses Courseplay? I know everyone thinks AI is going to rule the world but we players of Farming Simulator have looked at AI for years and well I'm not sure about the ruling the world bit. I have got to be honest even though most of Courseplay is algorithm driven, there are many aspects of using Courseplay that you can nudge if you've used it for long enough. I love creating the courses, can we expect your continued excellent Mod for FS25?

**Comment by Tensuko** [comment]
![grafik](https://github.com/user-attachments/assets/98b8e2bd-ab29-4318-aebc-71ba3843ae89)
 #3328
